apps/server-web/src/renderer/App.css

```
@@ -476,15 +511,19 @@ html.dark {
   background: transparent;
 }
+
 .drag-region-windows {
   -webkit-app-region: drag;
-  height: 30px; /* Adjust height as needed */
+  height: 30px;
+  /* Adjust height as needed */
   cursor: pointer;
 }
+
 .drag-region::hover {
```

Biome (2.4.4)

[error] Unexpected unknown pseudo-elements: hover

(lint/correctness/noUnknownPseudoElement)